packages/cli/src/ceramic-daemon.ts

```diff
@@ -120,50 +123,6 @@ export function makeCeramicConfig(opts: DaemonConfig): CeramicConfig {
   return ceramicConfig
 }
 
-/**
- * Takes a query object and parses the values to give them proper types instead of having everything
- * as strings
- * @param opts
- */
-function parseQueryObject(opts: Record<string, any>): Record<string, string | boolean | number> {
-  const typedOpts = {}
-  for (const [key, value] of Object.entries(opts)) {
-    if (typeof value == 'string') {
-      if (value[0] == '{') {
-        // value is a sub-object
-        typedOpts[key] = parseQueryObject(JSON.parse(value))
-      } else if (value === 'true') {
-        typedOpts[key] = true
-      } else if (value === 'false') {
-        typedOpts[key] = false
-      } else if (!isNaN(parseInt(value))) {
-        typedOpts[key] = parseInt(value)
-      } else {
-        typedOpts[key] = value
-      }
-    } else {
-      typedOpts[key] = value
-    }
-  }
-  return typedOpts
-}
-
-/**
- * Converts 'sync' option sent as a bool by outdated http-clients to the current format of an enum.
- * The old behaviors don't map directly to the new behaviors, so we take the best approximation.
- * TODO remove this once we no longer need to support clients older than v1.0.0
- * @param opts
- */
-function upconvertLegacySyncOption(opts: Record<string, any> | undefined) {
```

Comment:
Just as a double check: Are we sure we don't have any clients running a version older than v1.0.0?

Reply:
We are! Few months ago we waited until 90% of the nodes run 2.37 or something.